plugins/coral-plugin-facebook-auth/server/router.js

```diff
@@ -0,0 +1,20 @@
+module.exports = (router) => {
+
+  const {passport, HandleAuthPopupCallback} = require('services/passport');
+
+  /**
+   * Facebook auth endpoint, this will redirect the user immediatly to facebook
+   * for authorization.
+   */
+  router.get('/api/v1/auth/facebook', passport.authenticate('facebook', {display: 'popup', authType: 'rerequest', scope: ['public_profile']}));
```

Comment:
is there a way to pass config stuff into `plugins.json` so they don't have to edit code if they want to change the scope or something?